tests/test_validator.py

```diff
@@ -115,13 +116,18 @@ def setUpClass(cls):
             for i in range(2)])
 
 
-class TestBaseValidator(TestValidatorTemplate):
+class TestBaseValidator(_TestValidatorBase):
     @classmethod
     def setUpClass(cls):
         cls.validator = _TaskValidator(task_type=TaskType.classification,
             few_samples_thr=1, imbalance_ratio_thr=50, far_from_mean_thr=5.0,
             dominance_ratio_thr=0.8, topk_bins=0.1)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_has_enum_entries_in_environment(self):
+        for key in TaskType._member_names_:
+            self.assertTrue(key in Environment().validators.items, key)
```

Comment:
I would suggest creating the environment once instead of on every iteration, and using `assertIn`.

Reply:
Done